cli/bpmetadata/proto/bpmetadata_ui.proto

```diff
@@ -126,6 +127,23 @@ message DisplayVariable {
   // Optional.
   // Gen: manually-authored
   string boolean_group = 20; // @gotags: json:"booleanGroup,omitempty" yaml:"booleanGroup,omitempty"
+
+  // Alternate default value.
+  // This allows authors to define an alternative value for pre identified usecases such as security.
+  // Gen: manually-authored.
+  message AlternateDefault {
+    // Type of the alternate default.
+    enum AlternateType {
```

Comment:
Going forward if a new alternate type is needed, will we update this enum? Let's say if we want a default for ADC use case, would we created new alternate type?

Reply:
Yes and I am thinking ADC can also able to leverage or surface some these existing alternates